articles/architecture-scenarios/b2c/tenant-architecture.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Tenant Architecture
+description: Learn how to create Auth0 tenants for one or more environments
+toc: true
+topics:
+    - b2c
+    - ciam
+    - tenants
+contentType: concept
+useCase:
+  - tenant-architecture
+---
+# Tenant Architecture
+
+Everything at Auth0 begins with the creation of a tenant. The tenant is where you will configure everything related to your use of Auth0.
+
+You can (and are encouraged to) create more than one tenant. In addition to using different tenants for different environments (for example, you can have a tenant for development and a tenant for production), you can use different tenants to isolate your user domains (determining which user domains you'll need is an important step of the tenant architecture process).
+
+::: warning
+Tenant names cannot be changed or reused (even if the original tenant has been deleted). Make sure that you are happy with your names before you set up the tenants.
+:::
+
+## Design considerations
+
+When you begin the process of designing your tenant architecture, consider the following questions:
+
+1. What should the URL be when the Authorization Server (Auth0) presents a page to the user? You tenant name is part of the URL, and as part of Phase 1, you may be setting up [custom domains](#custom-domains).
+
+1. How will you support the Software Development Life Cycle? Do you have development, QA, and production environments, with each needing its own tenant? Do you have an additional staging environment, or are you working with a minimalist setup featuring just development and production environments?
+
+1. Are there any other projects that might impact your Auth0 integration? For example, if there are future projects in the pipeline, consider your setup carefully to minimize the likelihood that you'll have to migrate your existing tenant to another at that time.
+
+1. How do you make sure that all of your tenants are appropriately associated with your Auth0 contract? Auth0 has a [set of procedures](#associating-tenants) for you to follow if you want all of your tenants associated with each other.
+
+## Custom domains
```

Comment:
Again is this documentation constructed from the latest B2C Welcome provided? I notice the section is using numbered bullet points; the original B2C Welcome section ( https://docs.google.com/document/d/1wHEmRsPVciJJVuyssBH3RyO1mLYIGqLj34-vhVFEODU/edit#bookmark=id.cf7hf3wug4wi) was changed to favour non-numbered bullet points.